table/tables/cache.go

```diff
@@ -51,6 +51,7 @@ type cachedTable struct {
 	cacheData atomic.Value
 	handle    StateRemote
 	renewCh   chan func()
+	totalSize int64
```

Comment:
totalSize seems may not be very accurate? Because it is from read lock. It can be a stale value when last read cache is old. But I don't think its a big problem though.

Reply:
> totalSize seems may not be very accurate?

Yes, it's obtained when loading the table... and write will not update it, so it could be stale before the next loading